contributing-docs/03_a_contributors_quick_start.rst

```diff
@@ -0,0 +1,135 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+*************************
+Contributor's Quick Start
+*************************
+
+**The outline for this document in GitHub is available at top-right corner button (with 3-dots and 3 lines).**
+
+Note to Starters
+################
+
+Airflow is a complex project, as is setting up an Airflow development environment to test changes.
+This guide and its sub-guides are designed to make it as easy as possible to contribute.
+
+There are three main ways you can run an Airflow development environment:
+
+1. With `Breeze <../dev/breeze/doc/README.rst>`_ (local) Breeze...
```

Comment:
This reads weird, _With Breeze (local) Breeze..._

Reply:
+1